### ISSUE TITLE
Add more trailers....

### ISSUE THREAD

**Issue by mitchiemt11** [mitchiemt11/trailers-trove#4]
- The cool cats at Trailer Addict (https://www.traileraddict.com/trailerapi)
- The movie magic makers over at The Movie Database (https://developer.themoviedb.org/reference/movie-videos)
- The reel deal from MovieGlu (https://developer.movieglu.com/v2/api-index/trailers/)

The provided links for APIs are the next step in our movie trailer journey, and we'll incorporate them once we're thrilled with our current dummy data. So, let's keep having fun with the dummy data for now, and when we're ready, we'll roll out the red carpet for those APIs. Stay tuned for the blockbuster transformation! 🍿🎬🌟

**TL;DR**

**Your Task**

For now, we're spicing up our movie trailer display with some playful dummy data. Soon, we'll jazz it up even more by hooking into the latest movie trailers! 
We'll be adding these fresh trailers manually, making sure they're not a double feature with what's already in our data.js. And remember, the maximum number of trailers we're rolling out on each contributor is four. 
Enjoy the show! 🎥🍿

**Comment by Piyush07p** [comment]
can u assign it to me

**Comment by mitchiemt11** [comment]
@Piyush07p I have assigned this to you. Drop a PR when you are done. I will merge asap. Thanks!

**Comment by mitchiemt11** [comment]
@Piyush07p your PR LGTM. 

I'm reopening this issue for anyone to add more trailers before we start pulling trailers from API
